themes/grain/source/css/style.scss

```
@@ -515,6 +515,21 @@ table {
   }
 }
 
+details {
+  margin: 0.5rem 0;
+  font-size: 1rem;
+
+  &[disabled] {
+    > summary {
+      pointer-events: none;
+    }
+    > summary::marker,
+    > summary::-webkit-details-marker {
+      color: var(--gray2);
```

I also liked this a lot as the lighter gray (maybe a bit more). Up to you to take this suggestion!
```suggestion
      color: var(--gray);
```

---

I'll try it, but that gray looked not really gray in my editor.